cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -56,3 +77,94 @@ func (s *Server) GetClient(ctx context.Context) (dynamic.Interface, error) {
 	}
 	return client, nil
 }
+
+// GetManager ensures a manager is available and uses it to return the client.
+func (s *Server) GetManager() (assetManager, error) {
+	if s.manager == nil {
+		return nil, status.Errorf(codes.Internal, "server not configured with manager")
+	}
+	manager := s.manager
+	return manager, nil
+}
+
+// GetAvailablePackageSummaries returns the available packages based on the request.
+func (s *Server) GetAvailablePackageSummaries(ctx context.Context, request *corev1.GetAvailablePackageSummariesRequest) (*corev1.GetAvailablePackageSummariesResponse, error) {
+
+	contextMsg := ""
+	if request.Context != nil {
+		contextMsg = fmt.Sprintf("(cluster=[%s], namespace=[%s])", request.Context.Cluster, request.Context.Namespace)
+	}
+
+	log.Infof("+helm GetAvailablePackageSummaries %s", contextMsg)
+
+	namespace := ""
+	if request.Context != nil {
+		if request.Context.Cluster != "" {
+			return nil, status.Errorf(codes.Unimplemented, "Not supported yet: request.Context.Cluster: [%v]", request.Context.Cluster)
+		}
+		if request.Context.Namespace != "" {
```

Comment:
Ooops, just a leftover after copy-n-pasting from the kapp_controller plugin (there, it was initialized to `namespace := globalPackagingNamespace`  instead of  `""`). Thanks for pointing this out